pennylane/tape/jacobian_tape.py

```diff
@@ -446,6 +491,10 @@ def jacobian(self, device, params=None, **options):
             order=1 (int): The order of the finite difference method to use. ``1`` corresponds
                 to forward finite differences, ``2`` to centered finite differences.
             shift=pi/2 (float): the size of the shift for two-term parameter-shift gradient computations
+            argnum=None (int, list(int), None): Which argument(s) to compute the jacobian
+                with respect to. When there are fewer parameters specified than the
+                total number of trainable parameters, the jacobian is being estimated by
+                sampling ``argnum`` many parameters from the set of trainable parameters.
```

Comment:
Is this sampling thing still the case? Does this need to get updated?

Reply:
Good point, removed that part.